js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
 noDirectUsers=Keine direkten Benutzer
 permissionsDisable=Berechtigung abschalten?
 eventTypes.PERMISSION_TOKEN.name=Berechtigungstoken
+eventTypes.UPDATE_TOTP.name=Update totp
+subjectNameId=Betreff NameID
+serviceProviderEntityId=Service Provider Entity ID
+artifactBindingUrl=Artifact Binding URL
+eventTypes.LOGIN.name=Anmelden
+clearFileExplain=Sind Sie sicher, dass Sie diese Datei leeren möchten?
+eventTypes.LOGOUT_ERROR.name=Logout Fehler
```

Comment:
Similar to the above, I suggest for example `Fehler beim Anmelden` or `Anmeldefehler`. Please review the other `eventTypes.*.name` entries as well.